### PR TITLE
Ticket Editor Date formatting

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#91058](http://central.tri.be/issues/91058) [C#91077](http://central.tri.be/issues/91077) [C#91044](http://central.tri.be/issues/91044)
---

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $meta_id is never used | [Tickets_Handler.php:177](https://github.com/moderntribe/event-tickets/blob/c8931a9cff1d480fa2bc8cc17be33cbb346aa799/src/Tribe/Tickets_Handler.php#L177)
:warning: | The method parameter $date is never used | [Tickets_Handler.php:177](https://github.com/moderntribe/event-tickets/blob/c8931a9cff1d480fa2bc8cc17be33cbb346aa799/src/Tribe/Tickets_Handler.php#L177)
:warning: | The method parameter $meta_id is never used | [Tickets_Handler.php:259](https://github.com/moderntribe/event-tickets/blob/c8931a9cff1d480fa2bc8cc17be33cbb346aa799/src/Tribe/Tickets_Handler.php#L259)
:warning: | The method parameter $update is never used | [Tickets_Handler.php:303](https://github.com/moderntribe/event-tickets/blob/c8931a9cff1d480fa2bc8cc17be33cbb346aa799/src/Tribe/Tickets_Handler.php#L303)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Tickets.php [hunk 1]
```diff
@@ -923,7 +927,31 @@ public function ajax_refresh_settings() {
...
 			// Pass the control to the child object
-			return $this->save_ticket( $post_id, $ticket, $data );
+			$save_ticket = $this->save_ticket( $post_id, $ticket, $data );
+
+			tribe( 'tickets.handler' )->toggle_manual_update_flag( false );
+
+			$post = get_post( $post_id );
+			if ( empty( $data['ticket_start_date'] ) ) {
+				// 30 min in seconds
+				$round = 1800;
```
Comment (Camwyn) [inline src/Tribe/Tickets.php]: You know you can do `30 * MINUTE_IN_SECONDS` and save the comment, right?
https://codex.wordpress.org/Easier_Expression_of_Time_Constants

**Comment by Camwyn** [comment]
Should probably rename those params to `$unused_`, but not a blocker.

File: src/Tribe/Tickets.php (hunk 1 above)
Comment (Camwyn) [reply src/Tribe/Tickets.php]: I like using the constants whenever possible - for transparency (and to reduce the chance I typo a number ;) )

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $meta_id is never used | [Tickets_Handler.php:177](https://github.com/moderntribe/event-tickets/blob/256d260d689df88ff889a0df20ed147a4691f287/src/Tribe/Tickets_Handler.php#L177)
:warning: | The method parameter $date is never used | [Tickets_Handler.php:177](https://github.com/moderntribe/event-tickets/blob/256d260d689df88ff889a0df20ed147a4691f287/src/Tribe/Tickets_Handler.php#L177)
:warning: | The method parameter $meta_id is never used | [Tickets_Handler.php:259](https://github.com/moderntribe/event-tickets/blob/256d260d689df88ff889a0df20ed147a4691f287/src/Tribe/Tickets_Handler.php#L259)
:warning: | The method parameter $update is never used | [Tickets_Handler.php:303](https://github.com/moderntribe/event-tickets/blob/256d260d689df88ff889a0df20ed147a4691f287/src/Tribe/Tickets_Handler.php#L303)

> via jenkins/codesniffer-phpcs

**Pull request merged**